doc/content/source/postprocessors/PeriodicAmplitudeRegulator.md

```diff
@@ -0,0 +1,40 @@
+# PeriodicAmplitudeRegulator
+
+This Postprocessor is intended to be used to modify the value of a controll paramter to force another Postprocessor value to be equal a refence value. The value of the Postprocessor is updated with the following scheme
+
+\begin{equation}
+  V_n =
+  \frac{P_\text{ref}}{P_\text{actual}}
+  V_{n-1}
+\end{equation}
+
+Where $V_n$ is the new value of the PeriodicAmplitudeRegulator Postprocessor, $P_\text{ref}$ is the reference value, $P_\text{actual}$ is the actual calculated value of the Postprocessor specified by the parameter `value`, and $V_{n-1}$ is the pervious value of the PeriodicAmplitudeRegulator Postprocessor.
+
+
+
+!row!
+!col!  small=12 medium=6 large=6
+!style halign=center
+!media media/periodic_modification.png
+       style=width:100%;display:block;margin-top:3em;margin-left:auto;margin-right:auto;
+!col-end!
+
+!col! small=12 medium=6 large=6
+!style halign=right
+!media media/multi_period_mean.png
+       style=width:97%;display:block;margin-top:3em;margin-left:auto;margin-right:auto;
+!col-end!
+!row-end!
+
+
+The results in the figure above are from `test/tests/postprocessors/periodic_amplitude_regulator/periodic_modifier.i` with a more refined time step, `dt = 1e-2`.
+
+## Example Input File Syntax
+
+!listing test/tests/postprocessors/periodic_amplitude_regulator/periodic_modifier.i
```

Comment:
Limiting the Example Input File Syntax to just show the Functions and Postprocessors Blocks.

```suggestion
!listing test/tests/postprocessors/periodic_amplitude_regulator/periodic_modifier.i block=Functions Postprocessors
```